src/components/colorbar/draw.js

```diff
@@ -243,22 +245,38 @@ module.exports = function draw(gd, id) {
                 s.append('g').classed('cbtitleunshift',true)
                     .append('g').classed('cbtitle',true);
                 s.append('rect').classed('cboutline',true);
+                s.select('.cbtitle').datum(0);
             });
-        container.attr('transform','translate('+Math.round(fullLayout._size.l)+
-            ','+Math.round(fullLayout._size.t)+')');
+        container.attr('transform','translate('+Math.round(gs.l)+
```

Comment:
That would be pretty nice, but as a general rule I don't like to extend things that aren't mine - perhaps a util function? It would be pretty nice to have it return the transform as well with positions e.g. `{ x: 123, y: 45 }`.